Client/Application/ImageIdentifiers.swift

```diff
@@ -68,4 +68,5 @@ public struct ImageIdentifiers {
     public static let warning = "menu-warning"
     public static let whatsNew = "whatsnew"
     public static let xMark = "nav-stop"
+    public static let sponsoredStar = "sponsored-star"
```

Comment:
nit: ideally sponsoredStar would be ordered alphabetically

Reply:
Updated